src/pages/HomePage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import {Link} from 'react-router-dom';
+import  beers from '../assets/beers.png';
+import random from '../assets/random-beer.png';
+import newBeer from '../assets/new-beer.png';
+
+function HomePage() {
+  return (
+    <div>
+    <img src={beers}/>
+    <Link to="/beers"><h1>Beers</h1></Link>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras dignissim tincidunt lorem at finibus. Aliquam mi purus, pulvinar id odio eu, blandit dignissim ipsum. Integer finibus nulla id felis tempus vehicula. Quisque aliquet neque ac dapibus aliquam. Cras egestas fermentum sapien ac dictum. Quisque vel arcu odio. Maecenas elementum commodo ligula, id placerat enim congue id.</p>
+    
+    <img src={random}/>
+    <Link to="/random-beer"><h1>Random Beer</h1></Link>
```

Comment:
If you check on `App.js`, your random route is to `random`, so here it should be `random` as well. Or you can have `random-beer` here and change it there to `random-beer`